### ISSUE TITLE
hyperkit: `minikube ip` shows an IP even if VM is stopped

### ISSUE THREAD

**Issue by tstromberg** [kubernetes/minikube#3271]
As reported on #minikube by @dhs227. It's noteworthy that `status` shows that minikube is stopped:

```
[~/c/m/hellonode] : minikube status                                                                             10:40:28 PM
minikube: Stopped
cluster: 
kubectl: 
```

**Comment by tstromberg** [comment]
It's worth noting that this does not appear to affect VirtualBox.

**Comment by tstromberg** [comment]
Confirmed in v0.34.1

**Comment by tstromberg** [comment]
Apparently this was fixed in v1.0.
